src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -166,7 +166,7 @@ public boolean isAACLC() {
 	/**
 	 * @return True if the audio codec is AAC HE.
 	 */
-	public boolean isAACHE() {
+	public boolean isHEAAC() {
```

Comment:
Audio PR.
Your logic is...your ;)
The one in MediaInfo, FFmpeg...are following the IETF, IANA profile:
AAC-LC profile
HE-AAC profile

That's said, before seing that mess, i'll have choosen like you, or not modified anything.